butter_cms/content_field.py

```diff
@@ -12,3 +12,13 @@ def get(self, keys=None, params=None):
             params = {}
         params['keys'] = ','.join(keys)
         return self.api_get(params=params)
+    
+    def get_collection(self, slug, fields=None, params=None):
+        if not params:
+            params = {}
+        if fields:
+            for field in fields:
+                key = f"fields.{field}"
+                params[key] = fields[field]
+
+        return self.api_get(slug, params=params)
```

Comment:
Why was this added? I don't see it being used anywhere:
```
$ rg get_collection
butter_cms/content_field.py
16:    def get_collection(self, slug, fields=None, params=None):
```

Reply:
The method `get` is checking for all collections with ability to filter based on `keys` (this is used without `slug` - you can see uses of this in [docs](https://buttercms.com/docs/api) if you search for `content/?keys=` on the page), but there is no support for checking specific collection (docs: https://buttercms.com/docs/api/#filtering-a-collection) that is able to filter items in specific collection.

Reply:
Makes sense. I think we should add unit tests for that, tho. I think we also should consider adding docstrings (those could be added for everything in the SDK tbh, so maybe we should consider this as a separate task and PR cc @ViolanteCodes).